sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -720,14 +720,6 @@ object SQLConf {
     .stringConf
     .createWithDefault("_corrupt_record")
 
-  val FROM_JSON_FORCE_NULLABLE_SCHEMA = buildConf("spark.sql.fromJsonForceNullableSchema")
```

Comment:
Can we throw an exception if users try to set the removed conf to a value that is different from the default?

Reply:
Sure, we could throw an exception for 3 configs.  I am just wondering why we silently ignore non-existed SQL configs:
```scala
scala> spark.conf.set("spark.sql.abc", 1)

```
How about throwing `AnalysisException` for not existed SQL configs that have the `spark.sql` prefix but don't present in https://github.com/apache/spark/blob/6d64fc2407e5b21a2db59c5213df438c74a31637/sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala#L50 ?

or there are SQL configs that we have to bypass for some reasons?

Reply:
Here is the PR https://github.com/apache/spark/pull/27057